### DISCUSSION TITLE
media queries

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#61] [Q&A] [answered]
What is the use of media queries in CSS3?

**Comment by Bunny77K9** [comment, marked as answer]
Media queries are used to define styles in CSS, which are responsive based on a variety of shapes and sizes of the viewing window.

They are used to adjust the following entities:

    Height
    Width
    Viewport
    Resolution
    Orientation

If you are looking forward to becoming proficient in [Web Development](https://intellipaat.com/blog/what-is-web-development/), make sure to check out Intellipaat’s latest offerings for [Full Stack Developer Course](https://intellipaat.com/full-stack-web-developer-mean-stack-certification-training/).
